astropy/utils/system_info.py

```diff
@@ -0,0 +1,84 @@
+r"""
+A helper script to automate system info in bug reports.
+
+This can be run as
+```
+python -c "import astropy; astropy.system_info()"
+```
```

Comment:
I think triple backticks is markdown, not RST. 
```suggestion
This can be run as::

    python -c "import astropy; astropy.system_info()"

```